services/src/main/java/org/exoplatform/webconferencing/externalvisio/service/impl/ExternalVisioConnectorServiceImpl.java

```diff
@@ -31,18 +34,24 @@
 public class ExternalVisioConnectorServiceImpl implements ExternalVisioConnectorService {
 
   private final ExternalVisioConnectorDAO externalVisioConnectorDAO;
+  
+  private final ProfilePropertyService profilePropertyService;
 
-  public ExternalVisioConnectorServiceImpl(ExternalVisioConnectorDAO externalVisioConnectorDAO) {
+  public ExternalVisioConnectorServiceImpl(ExternalVisioConnectorDAO externalVisioConnectorDAO,
+                                           ProfilePropertyService profilePropertyService) {
     this.externalVisioConnectorDAO = externalVisioConnectorDAO;
+    this.profilePropertyService = profilePropertyService;
 
   }
 
   @Override
-  public ExternalVisioConnector createExternalVisioConnector(ExternalVisioConnectorEntity externalVisioConnectorEntity) {
+  public ExternalVisioConnector createExternalVisioConnector(ExternalVisioConnectorEntity externalVisioConnectorEntity) throws ObjectAlreadyExistsException {
```

Comment:
why throwing the error here ?
if the setting property exists, so it is ok, catch the error, log a warn, and continue.
No need to propagate the error